tools.go

```diff
@@ -0,0 +1,12 @@
+//go:build tools
+// +build tools
+
```

Comment:
1. Why do we need this file
2. Where do we use the `tools` build directive?

Reply:
1. We need it so that running `go mod tidy` does not delete our linter deps. [Context](https://www.reddit.com/r/golang/comments/10rlp31/toolsgo_pattern_still_valid_today_i_want_to/). It also exists in RDK
2. I will delete. This file was cribbed from RDK and I don't think we have that build tag on this repo

Reply:
Understood, thanks!

Reply:
Actually ```The build constraints //go:build tools and //go:build tools,!main are used to mark packages that are part of the "tools" build tag. This tag is used to identify packages that are part of the development toolchain, rather than the main application.```

I will keep them

Reply:
@nicksanford ^ re: resolving the build flag in tools.go